packages/labs/analyzer/src/lib/model.ts

```diff
@@ -29,6 +34,28 @@ export class Package {
     this.tsConfig = init.tsConfig;
     this.modules = init.modules;
   }
+
+  getModulesWithDeclarations<T extends Declaration>(
```

Comment:
comment?

Reply:
Removed, and commented `getLitElementModules`